### ISSUE TITLE
minishift config set iso-url minikube doesnt work

### ISSUE THREAD

**Issue by MansM** [minishift/minishift#1982]
### General information

  * Minishift version:  v1.12.0+daa0943
  * OS: macOS High Sierra 
  * Hypervisor: VirtualBox

### Steps to reproduce

  1.  minishift config set iso-url minikube
  2. 
  3. 
  4. 

### Expected
config updated with minikube as iso setting

### Actual
minikube url is not valid: parse minikube: invalid URI for request]

Putting it directly in the config file (see below)  or  as an option during start works like a charm

cat ~/.minishift/config/config.json
```
{
    "addons": {
        "admin-user": {
            "Enabled": true,
            "Name": "admin-user",
            "Priority": 0
        }
    },
    "cpus": 4,
    "image-caching": true,
    "iso-url": "minikube",
    "memory": "8G",
    "openshift-version": "v3.7.1",
    "vm-driver": "virtualbox"
}
```

**Comment by praveenkumar** [comment]
@MansM Thanks for filling this issue, looks like we missed to add our whitelist in term of isos during the refactor :(

**Comment by MansM** [comment]
reading the code it aint that hard to fix, will create a pr shortly

**Comment by praveenkumar** [comment]
resolved by #1984
